intlist.h

```diff
@@ -0,0 +1,42 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// intlist.h
+// Program on main branch.
+// ------------------------------------------------------------------------------------------------
+/*
+Aim : To implement a linked list data structure to handle integer elements.
+The implementation consists of three files:
+              intlist.h              -             the header file
+              intlist.c              -             the code.
+              testintlist.c          -             the program that tests the implementation.
+The following functions have been implemented:
+a)	Create �  creates a new linked list.
+b)	Delete �  deletes the list and all the linked elements.

```

Comment:
There is some strange character here. � What is it?

Reply:
I used hyphens in those places. I have changed it to semi-colons. Thank you.